src/DataTable/Table.js

```diff
@@ -64,11 +81,20 @@ class Table extends React.Component {
                     </tr>
                 </thead>
                 <tbody>
-                    {realRows.map((row, idx) =>
-                        <tr key={row[rowKeyColumn] || row.key || idx} className={row.className}>
-                            {columnChildren.map((child) => this.renderCell(child.props, row, idx))}
-                        </tr>
-                    )}
+                    {realRows.map((row, idx) => {
+                        const customRowProps = getCustomRowProps(row);
+
+                        return (
+                            <tr
+                                key={row[rowKeyColumn] || row.key || idx}
+                                style={customRowProps.style}
+                                className={customRowProps.className}
```

Comment:
You're right, that's a pretty big miss. Sorry about that.

Which one would you favor? Adding to mdlRowProps (creating it in the process) or merging two separate objects?

I'd tend to favor keeping the current behaviour, only adding more classNames to it via mdlRowProps, in case people like me were already overloading row.className. What do you think?